client/Log.cpp

```diff
@@ -34,11 +34,11 @@ std::string FormatIP4Address(unsigned long ip)
 	return ss.str();
 }
 
-std::string FormatPlayerList(const PeerInfo* peerInfos)
+std::string FormatPlayerList(const std::array<PeerInfo, 6> peerInfos)
```

Comment:
I assume you probably wanted to pass this by reference rather than by value.

----

And also the hardcoded constant.

Reply:
Oh yeah, good catch. Having to follow Array around with the hardcoded constant makes me want to use vector instead.

Reply:
You could potentially use a type alias to shorten it:
```cpp
using PlayerList = std::array<PeerInfo, MaxRemotePlayers>;
```

Though I tend to prefer avoiding such type aliases until they get much longer, or use function pointer syntax.